### PR TITLE
Address comments in issue 72

### PR DESCRIPTION
**Pull request by ved-rivos**

### CODE REVIEW

**Review by eckhard-delfs-qualcomm — COMMENTED** [review]
Thanks for adding the example.
(In case ranges sizes of only up to 4GB should be expressed here I think PPN would need to be restricted to [20:0].)

I will quickly feedback on the "limit/restrict" issue.

File: chapter6.adoc [hunk 1]
```diff
@@ -489,3 +489,23 @@ This range is decided by the low-order bits of the `PPN` field, going up to the
 first low-order 0 bit (inclusive of this position). If the initial low-order 0
 bit position is denoted as `x`, the size of the range is computed as
 `(1 << (12 + x + 1))`.
+
+[NOTE]
+====
+The following example illustrates the use of `S` field to specify an address
+range for the `MTTINVAL` operation. The example shows encoding ranges of up to
+4 GiB. Larger ranges may be encoded using the upper address bits (bits 43:22)
```
[the remark this answers is not in the document]
Comment (ved-rivos) [reply chapter6.adoc:497]: updated to fix the example.

**Comment by eckhard-delfs-qualcomm** [comment]
> Thanks for adding the example. (In case ranges sizes of only up to 4GB should be expressed here I think PPN would need to be restricted to [20:0].)
> 
> I will quickly feedback on the "limit/restrict" issue.

Our feedback has been covered by Andrew in issue https://github.com/riscv/riscv-smmtt/issues/72#issuecomment-2388626093

**Review by eckhard-delfs-qualcomm — APPROVED** [review]
looks good to me - thanks!

**Pull request merged**